packages/react-components/src/components/video/progress-bar.tsx

```diff
@@ -11,6 +11,9 @@ const Bar = styled.div<{ barHeight: number }>`
     bottom: 0;
     left: 0;
     opacity: 0.95;
+    @media only percy {
```

Comment:
another thing we need to hide when visual diffing in percy, the progress bar was always at a different length